Docs/Configuration.tex

```diff
@@ -575,31 +572,31 @@ \subsection{Installation and Upgrade}\label{configuration-install}
 the \hyperref[miscsecurityprops]{Security Properties} section of this document.
 
 The \texttt{OC\ config} file, as with any property list file, can be edited with
-any text editor such as nano and vim. However, specialised software may provide
-a better experience. On macOS, the preferred GUI application is
+any text editor such as nano, vim, TextEdit or NotePad. However, specialised software
+may provide a better experience. On macOS, the preferred GUI application is
```

Comment:
This is getting very nit-picky, sorry - but I still don't see the need to add those two!

nano and vim are perfectly good as examples of text editors.

To be clear, I am reading "such as" as a perfectly good English replacement for "e.g.".

Therefore I don't think you need four examples? (And also, using by far the two most widely used classic Unix editors as examples seems preferable to necessarily more arbitrarily picking a couple of the author's preferred other ones!)

Suggest:

```
any text editor, such as nano or vim.
```

Reply:
Will never be able to get 100% agreement.

I think adding the two is important so leaving them in.

Reply:
Could also swap "nano and vim" for "TextEdit and NotePad" if there is a strong preference for two items.

Reply:
cc @vit9696

Reply:
I know you've been kind enough to already make a change in response to my comment - but I would definitely notice and stop short if I was reading _four_ examples in the docs (i.e. just as a normal user). I mean, _why_? And why _those_?! (Would be what I would think.) So Vit review?

Reply:
> Could also swap "nano and vim" for "TextEdit and NotePad" if there is a strong preference for two items as I feel the latter duo better represent the idea of "any text editor".
>
> There might however be a desire to "look down" on those two in which case, just having nano and vim is perfectly fine as you pointed out.

I might well be looking down (!) (not that I don't also use them, too!) but also, as stated, they are known, long standing, classic options - essentially everything else is an arbitrary choice between different, competing, newer options (or very much more obscure older options). And it would seem that mentioning those two command line editors is much more in keeping with the spirit of OC than abitrary higher level, GUI specific editors.

Reply:
Not so sure either of us is particularly placed to articulate the "Spirit of OC" and the categorisation of the two most widely know text editors as arbitrary is unfortunate.

Either way, gone further and actually replaced nano and vim with those two.

Thanks though for the very helpful and useful review.

Vit always will review so actually.

Reply:
Oh dear - oh well, up for review now anyway!

Thank you for the thanks - very glad to have been able to offer input.

Reply:
(Two more quick things, sorry: It's Notepad, not NotePad - assuming it's the Windows one - also I don't know the current status, but historically Notepad was not great with line endings.... just thought of this, might matter?)